shared/ui-components/src/page_components/UserNav.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react"
+import "./UserNav.scss"
```

Comment:
I don't see a connection between what's in the CSS getting imported and this component?

Reply:
Good catch 👍